back/requirements.txt

```diff
@@ -0,0 +1,10 @@
+Flask==2.3.2
+gunicorn==20.1.0
+python-dotenv==0.19.2
+Authlib==0.15.5
+google-cloud-logging
+google-cloud-ndb
+google-cloud-recaptcha-enterprise
+google-cloud-secret-manager
+absl-py
+flask-cors
```

Comment:
Please specify versions for these so we don't accidentally get breaking changes.